cmd/adsysd/integration_tests/adsys_test.go

```diff
@@ -257,7 +264,9 @@ apparmor_dir: %s/apparmor.d/adsys
 apparmorfs_dir: %s/apparmorfs
 systemunit_dir: %s/systemd/system
 global_trust_dir: %s/share/ca-certificates
-`, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.backend, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir))
+
+detect_cached_ticket: %t
+`, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.backend, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.adsysDir, args.detectCachedTicket))
```

Comment:
one more, one more! (yeah, this is starting to grow… uncontrolled, but I don’t have a good solution TBH)